src/_pyinstaller_hooks_contrib/hooks/stdhooks/hook-gmplot.py

```diff
@@ -0,0 +1,15 @@
+# ------------------------------------------------------------------
+# Copyright (c) 2020 PyInstaller Development Team.
+#
+# This file is distributed under the terms of the GNU General Public
+# License (version 2.0 or later).
+#
+# The full license is available in LICENSE.GPL.txt, distributed with
+# this software.
+#
+# SPDX-License-Identifier: GPL-2.0-or-later

```

Comment:
And we use `Apache-2.0` identifier (not that I know what the difference is). Just copy paste from one of the other files.